pom.xml

```diff
@@ -132,7 +132,7 @@
   <repositories>
     <repository>
       <id>jetbrains-all</id>
-      <url>https://download.jetbrains.com/teamcity-repository</url>
+      <url>http://download.jetbrains.com/teamcity-repository</url>
```

Comment:
@pavel-github please revert this single line :)

Reply:
you're right :), after rebasing feature branch I don't need to revert this commit.